test/controllers/project_skill_controller_test.exs

```diff
@@ -71,75 +60,49 @@ defmodule CodeCorps.ProjectSkillControllerTest do
 
       insert(:organization_membership, role: "admin", member: current_user, organization: organization)
```

Comment:
No, in the latter the `admin` is the "site admin".  In the former the admin is the admin role on the organization.

I think we need to open an issue to change `@tag authenticated: :admin` to `@tag authenticated: :site_admin` for more explicitness.